recipes/bzip2/build.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 set -eu
 
-mkdir -p $PREFIX
-cp -r include $PREFIX
-cp -r lib $PREFIX
+mkdir -p "$PREFIX"
+cp -r include "$PREFIX"
+cp -r lib "$PREFIX"
 
 # Strip out any dylib files to ensure static linking
-find $PREFIX -name "*.dylib" -exec rm -rf {} \;
+find "$PREFIX" -name '*.dylib' -exec rm -rf {} \;
```

Comment:
So, i presume the intention here is for the `find` command to ingest the asterisk. If so, I've added single quotes so the shell doesn't evaluate the asterisk.

Note: applies to several other recipes as well.

Reply:
Correct - this is purging any `.dylibs` from the packaged wheel so that they're not found during linking, resulting in a binary that will try to load the dynamic form of the library at runtime.

Reply:
Neither zsh nor bash evaluates wildcards inside quotes, either double or single.

Reply:
Yeah, you're absolutely right. After writing this, it became clear my previous understanding about this handling wasn't correct; TIL :)

That said, this brings up another point; several of these scripts are only compatible with Bash. If we change the shebangs to `/bin/sh`, shellcheck will instead target much more universal shell code. This is probably mostly an issue for the `setup-iOS.sh` script but a few others places may be affected as well.

Reply:
> If we change the shebangs to `/bin/sh`, shellcheck will instead target much more universal shell code.

I think that would be a good idea, because macOS comes with an extremely old version of Bash from 2007, which they will never update because they're scared of the GPL3.